### PR TITLE
feat(AC): add current UI for Vue

### PR DESCRIPTION
**Pull request by kensodemann**

### CODE REVIEW

File: docs/native-solutions/auth-connect/vue/popup-vs-current.md [hunk 1]
```diff
@@ -0,0 +1,563 @@
...
+sidebar_position: 4
+---
+
+## Overview
+
+When the application is running in a web context, Auth Connect provides two different options for presenting the authentication page: `popup` or `current`. Up to this point, we have been using `popup`. In this tutorial we will explore `current`.
+
+### `popup`
+
+With `popup`, the authentication provider is opened in a new browser tab / window. This mode is the most consistent with how Auth Connect works on mobile where the authentication provider is displayed in a secure web view. On the web, this  This option requires no extra code, but it may not be the best user experience for web.
```
Comment (dtarnawsky) [inline docs/native-solutions/auth-connect/vue/popup-vs-current.md:13]: "On the web, this This option" should be "On the web, this option"

File: docs/native-solutions/auth-connect/vue/popup-vs-current.md [hunk 2]
```diff
@@ -0,0 +1,563 @@
...
+
+If your application is only distributed as a web-native mobile app, and the web-context is only used for development, then it is best to use `popup`.
+ 
+### `current`
+
+With `current`, the authentication provider is opened in the current windows, replacing your application. Your application will then be called with token information on the URL upon successful login. Since this is fundamentally different than the mobile implementation, it also means that special code is needed to handle it.
```
Comment (dtarnawsky) [inline docs/native-solutions/auth-connect/vue/popup-vs-current.md:19]: "in the current windows" > "in the current window"

Comment (kensodemann) [reply docs/native-solutions/auth-connect/vue/popup-vs-current.md:19]: removed 's'

File: docs/native-solutions/auth-connect/vue/popup-vs-current.md [hunk 3]
```diff
@@ -0,0 +1,563 @@
...
+
+If your application is distributed in a web context, it is worth considering using `current` for an improved user experience on web.
```
[the remark this answers is not in the document]
Comment (kensodemann) [reply docs/native-solutions/auth-connect/vue/popup-vs-current.md:21]: Fixed.

**Review by dtarnawsky — APPROVED** [review]

**Review by jameson-w-taylor — CHANGES_REQUESTED** [review]

File: docs/native-solutions/auth-connect/vue/popup-vs-current.md [hunk 4]
```diff
@@ -0,0 +1,563 @@
...
+
+### `popup`
+
+With `popup`, the authentication provider is opened in a new browser tab / window. This mode is the most consistent with how Auth Connect works on mobile where the authentication provider is displayed in a secure web view. On the web, this option requires no extra code, but it may not be the best user experience for web.
+
+If your application is only distributed as a web-native mobile app, and the web-context is only used for development, then it is best to use `popup`.
+ 
+### `current`
+
+With `current`, the authentication provider is opened in the current window, replacing your application. Your application will then be called with token information on the URL upon successful login. Since this is fundamentally different than the mobile implementation, it also means that special code is needed to handle it.
```
[the remark this answers is not in the document]
Comment (kensodemann) [reply docs/native-solutions/auth-connect/vue/popup-vs-current.md]: How about changing "called" to "restarted" and leaving the rest fundamentally the same?

File: docs/native-solutions/auth-connect/vue/popup-vs-current.md [hunk 5]
```diff
@@ -0,0 +1,563 @@
...
+With this code:
+
+- The `login()` is called and Auth Connect opens the OIDC authentication provider in a new tab (web) or a secure web view (mobile).
+- Auth Connect listens for that tab or secure web view to close.
+- If the user successfully logs in:
+   - Auth Connect unpacks the data sent back when the tab or secure web view is closed and creates an `AuthResult`.
+   - Our `login()` stores the `AuthResult` and resolves.
+- If the user cancels the operation, our `login()` rejects with an error.
+
+With `current`, this code will still work like this on mobile. However, on web our app is completely replaced by the
```
[the remark this answers is not in the document]
Comment (kensodemann) [reply docs/native-solutions/auth-connect/vue/popup-vs-current.md:57]: Updated as suggested.

File: docs/native-solutions/auth-connect/vue/popup-vs-current.md [hunk 6]
```diff
@@ -0,0 +1,563 @@
...
+    }
+  },
+});
+```
+
+</CH.Code>
+
+If search parameters are present, the this is the login returning. We package the data and send it to Auth Connect to process and create an `AuthResult`.
+
+If there are no parameter, we will assume a logout and set the `authResult` to `null`.
```
[the remark this answers is not in the document]
Comment (kensodemann) [reply docs/native-solutions/auth-connect/vue/popup-vs-current.md:450]: fixed

File: docs/native-solutions/auth-connect/vue/popup-vs-current.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jameson-w-taylor) [reply docs/native-solutions/auth-connect/vue/popup-vs-current.md]: I think that would be fine

**Review by jameson-w-taylor — APPROVED** [review]

**Pull request merged**